app/screens/post_options/post_options.tsx

```diff
@@ -3,7 +3,9 @@
 
 import {useManagedConfig} from '@mattermost/react-native-emm';
 import React from 'react';
+import {ScrollView} from 'react-native';
```

Comment:
From gesture handler, also how many items max could be added? If no limit ScrollView is not the way to go as is not opimized for perf

Reply:
On the order of O(n) apps installed, probably less then n. This is a temporary solution until we have a separate list for actions